distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/Client.java

```diff
@@ -13,121 +13,102 @@
  */
 package org.codice.alliance.nsili.client;
 
+import static com.xebialabs.restito.builder.stub.StubHttp.whenHttp;
+import static com.xebialabs.restito.semantics.Condition.method;
+
+import java.net.MalformedURLException;
+
 import org.codice.alliance.nsili.common.GIAS.PackageElement;
-import org.codice.alliance.nsili.common.GIAS.Query;
 import org.codice.alliance.nsili.common.NsilCorbaExceptionUtil;
-import org.codice.alliance.nsili.common.NsiliConstants;
 import org.codice.alliance.nsili.common.UCO.DAG;
-import org.codice.alliance.nsili.common.UID.Product;
-import org.glassfish.grizzly.http.server.HttpServer;
-import org.glassfish.grizzly.http.server.NetworkListener;
-import org.omg.PortableServer.POA;
-import org.omg.PortableServer.POAHelper;
+import org.glassfish.grizzly.http.Method;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.sun.jersey.api.container.grizzly2.GrizzlyServerFactory;
-import com.sun.jersey.api.core.PackagesResourceConfig;
-import com.sun.jersey.api.core.ResourceConfig;
+import com.xebialabs.restito.semantics.Action;
+import com.xebialabs.restito.semantics.Condition;
+import com.xebialabs.restito.server.StubServer;
 
 public class Client {
-    private HttpServer server = null;
-
-    public static int LISTEN_PORT = 8200;
 
     private static final Logger LOGGER = LoggerFactory.getLogger(Client.class);
 
-    private static final boolean SHOULD_PROCESS_PKG_ELEMENTS = false;
+    private static final int LISTEN_PORT = 8200;
```

Comment:
I don't think so. The only time that it's ever run with that port is from the command line by developers, and if you're running it there, you'll probably also be changing `SHOULD_PROCESS_PKG_ELEMENTS`, `SHOULD_TEST_STANDING_QUERY_MGR`, and `SHOULD_DOWNLOAD_PRODUCT`. I think simpler is better in this case, but I'm not set on that.